randovania/games/dread/presets/starter_preset.rdvpreset

```diff
@@ -3,7 +3,7 @@
     "base_preset_uuid": null,
     "name": "Starter Preset",
     "uuid": "407e97ed-decb-4889-8ab7-943f86ae48a4",
-    "description": "Basic preset.",
+    "description": " beginner-friendly preset for Metroid Dread featuring full item randomization, moderate difficulty, and quality of life improvements like skippable cutscenes and optional hints.",
```

Comment:
This should probably be "**A** beginner-friendly preset..."

Reply:
Sorry for my mistake. I fixed it.